samples/star/src/main/kotlin/com/slack/circuit/star/repo/PetRepository.kt

```diff
@@ -33,8 +33,8 @@ class PetRepositoryImpl @Inject constructor(private val petFinderApi: PetfinderA
   PetRepository {
   private lateinit var animals: List<Animal>
 
-  override suspend fun getAnimals(): List<Animal> {
-    if (!this::animals.isInitialized) fetchAnimals()
+  override suspend fun getAnimals(forceRefresh: Boolean): List<Animal> {
+    if (!this::animals.isInitialized || forceRefresh) fetchAnimals()
```

Comment:
this won't work right? Don't you need to make `animals` no longer a `lateinit var`?

Reply:
I ran this by Kier because I was also skeptical if this made sense, but it does still work. What would be the reason to not make `animals` a `lateinit var`.

Reply:
Adding a video to the PR desc.

Reply:
lateinit var can only be set once as far as I know, I thought they threw an exception if you set it multiple times